lib/data/external/GCP/GcpUtils.js

```diff
@@ -1 +1,62 @@
-module.exports = {};
+const werelogs = require('werelogs');
+const { errors } = require('arsenal');
+
+const _config = require('../../../Config').config;
+const { logHelper } = require('../utils');
+
+werelogs.configure({
+    level: _config.log.logLevel,
+    dump: _config.log.dumpLevel,
+});
+
+const logger = new werelogs.Logger('gcpUtil');
+
+class JsonError extends Error {
+    constructor(type, code, desc) {
+        super(type);
+        this.code = code;
+        this.description = desc;
+        this[type] = true;
+    }
+}
+
+function jsonRespCheck(err, resp, body, method, callback) {
+    if (err) {
+        logHelper(logger, 'error',
+            `${method}: error in json method`,
+            errors.InternalError.customizeDescription('json method failed'));
```

Comment:
Should we be passing the `dataStoreName` to this log helper function? https://github.com/scality/S3/blob/07245ac181a7e89deb1571ded058b33f5a024bf3/lib/data/external/utils.js#L25